project/app/src/main/java/com/fillr/example/integration/activity/ExampleWebViewHeadlessActivity.java

```diff
@@ -165,18 +193,9 @@ protected void onPause() {
         }
     }
 
-    /**
-     * Optional configuration. Used to identify when the hosting activity resumes.
-     */
-    private void fillrOnResume() {
-        if (fillr != null) {
-            fillr.onResume();

```

Comment:
We no longer need to know activity on resume any more?

Reply:
We do! It's in the lifecycle onResume method.